### ISSUE TITLE
(format "%%%s" "foo") reports mismatching argument count

### ISSUE THREAD

**Issue by mattiuusitalo** [borkdude/clj-kondo#1007]
**version**

clj-kondo v2020.07.29

**macro usage**

n/a

**platform**

native

**editor**

emacs

**problem**

(format "%%%s" "foo")
reports

"Format string expects 0 arguments instead of 1"

**repro**

(format "%%%s" "foo")

repro.clj:1:9: error: Format string expects 0 arguments instead of 1.
linting took 17ms, errors: 1, warnings: 0

**config**

n/a

**expected behavior**

It shouldn't report an error

This looks like something I could maybe pitch a PR in.

**Comment by borkdude** [comment]
@mattiuusitalo Thank. You can suppress this warning with 2020.09.09 using:

``` 
#_{:clj-kondo/ignore[:format]}
(format "%%%s" "foo")
``` 

I'll make sure this gets fixed in the next version.
